client/src/components/pages/home/PastTourneyCard.js

```diff
@@ -0,0 +1,52 @@
+import { Image, Typography } from "antd";
+import Text from "antd/lib/typography/Text";
+import React, { useState, useEffect } from "react";
+
+import ContentManager from "../../../ContentManager";
+
+import "./PastTourneyCard.css";
+import { Link } from "@reach/router";
+
+/**
+ * @typedef tourney
+ * @property {string} displayName
+ * @property {string} code
+ * @property {number} year
+ * @property {string} codeAndDivision
+ * @property {string} fullCode
+ */
+/**
+ * @param {object} props
+ * @param {tourney} props.tourney
+ */
+export default function PastTourneyCard({ tourney }) {
+  const [tourneyContent, SetTourneyContent] = useState(null);
+
+  useEffect(() => {
+    ContentManager.get(tourney.fullCode).then((content) => {
+      SetTourneyContent(content);
+    });
+  }, [tourney.fullCode]);
+
+  return (
+    <Link to={`/${tourney.year}/${tourney.codeAndDivision}/home`}>
+      <div className="u-rounded-border PastTourneyCard-container">
+        <Image
+          className="u-rounded-border PastTourneyCard-thumbnail"
+          src={`/public/backgrounds/${tourney.fullCode}.png`}
+          fallback="/public/gts-tournament-no-image.png"
+          preview={false}
+        />
+        <div className="PastTourneyCard-text-container">
+          <Typography className="PastTourneyCard-title">
+            <Text className="u-xxbold">{tourneyContent?.name || "..."}</Text>
```

Comment:
one inconsistency: the ongoing tourney titles are all caps+italic with the year also bold, whereas the past tourneys are title cased, non-italic, with the year not bold

these styles should probably be consistent. i personally prefer what you've done here for past tourneys